### ISSUE TITLE
Symfony property access and validator components required in composer.json

### ISSUE THREAD

**Issue by anuri73** [overblog/GraphQLBundle#672]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| BC Break report? | yes
| RFC?             | no
| Version/Branch   | 0.13.1

Installation from scratch with symfony5 throw an error :

```
Executing script cache:clear [KO]
 [KO]
Script cache:clear returned with error code 255
!!  Symfony\Component\ErrorHandler\Error\ClassNotFoundError {#6181
!!    #message: """
!!      Attempted to load class "PropertyAccess" from namespace "Symfony\Component\PropertyAccess".\n
!!      Did you forget a "use" statement for another namespace?
!!      """
!!    #code: 0
!!    #file: "./vendor/overblog/graphql-bundle/src/Transformer/ArgumentsTransformer.php"
!!    #line: 40
!!    trace: {
!!      ./vendor/overblog/graphql-bundle/src/Transformer/ArgumentsTransformer.php:40 { …}
!!      Overblog\GraphQLBundle\Transformer\ArgumentsTransformer->__construct() {}
!!      ./vendor/symfony/dependency-injection/ContainerBuilder.php:1088 { …}
!!      ./vendor/symfony/dependency-injection/ContainerBuilder.php:595 { …}
!!      ./vendor/symfony/dependency-injection/ContainerBuilder.php:1211 { …}
!!      ./vendor/symfony/dependency-injection/ContainerBuilder.php:1159 { …}
!!      ./vendor/symfony/dependency-injection/ContainerBuilder.php:1061 { …}
!!      ./vendor/symfony/dependency-injection/ContainerBuilder.php:595 { …}
!!      ./vendor/symfony/dependency-injection/ContainerBuilder.php:1211 { …}
!!      ./vendor/symfony/dependency-injection/ContainerBuilder.php:1159 { …}
!!      ./vendor/symfony/dependency-injection/ContainerBuilder.php:1562 { …}
!!      ./vendor/symfony/dependency-injection/ContainerBuilder.php:1113 { …}
!!      ./vendor/symfony/dependency-injection/ContainerBuilder.php:595 { …}
!!      ./vendor/symfony/dependency-injection/ContainerBuilder.php:1211 { …}
!!      ./vendor/symfony/dependency-injection/ContainerBuilder.php:1159 { …}
!!      ./vendor/symfony/dependency-injection/ContainerBuilder.php:1562 { …}
!!      ./vendor/symfony/dependency-injection/ContainerBuilder.php:1113 { …}
!!      ./vendor/symfony/dependency-injection/ContainerBuilder.php:595 { …}
!!      ./vendor/symfony/dependency-injection/ContainerBuilder.php:573 { …}
!!      ./vendor/symfony/dependency-injection/ContainerBuilder.php:541 { …}
!!      ./vendor/overblog/graphql-bundle/src/DependencyInjection/Compiler/TypeGeneratorPass.php:17 { …}
!!      ./vendor/symfony/dependency-injection/Compiler/Compiler.php:91 { …}
!!      ./vendor/symfony/dependency-injection/ContainerBuilder.php:734 { …}
!!      ./vendor/symfony/http-kernel/Kernel.php:518 { …}
!!      ./vendor/symfony/http-kernel/Kernel.php:126 { …}
!!      ./vendor/symfony/framework-bundle/Console/Application.php:169 { …}
!!      ./vendor/symfony/framework-bundle/Console/Application.php:75 { …}
!!      ./vendor/symfony/console/Application.php:140 { …}
!!      ./bin/console:42 {
!!        › $application = new Application($kernel);
!!        › $application->run($input);
!!        › 
!!        arguments: {
!!          $input: Symfony\Component\Console\Input\ArgvInput {#3 …}
!!        }
!!      }
!!    }
!!  }
!!  2020-04-02T13:11:23+00:00 [critical] Uncaught Error: Class 'Symfony\Component\PropertyAccess\PropertyAccess' not found
!!  
Script @auto-scripts was called via post-update-cmd

Installation failed, reverting ./composer.json to its original content.
```

**Comment by murtukov** [comment]
@anuri73 Just install the required dependency with `composer`.

Here is the command: `composer require symfony/property-access` 

This is a bug and shall be fixed in the future.

**Comment by mcg-web** [comment]
Fixed in [v0.13.2](https://github.com/overblog/GraphQLBundle/releases/tag/v0.13.2).
